MockDuck/Sources/MockRequestResponse.swift

```diff
@@ -107,6 +107,10 @@ final class MockRequestResponse: Codable {
             hashData.append(body)
         }
 
+        if let bodyData = normalizedRequest.httpBodyStreamData {
```

Comment:
Will this break any existing recordings?

Reply:
This will change the hash for the requests that have something in the .httpBodyStream. We can tag and release a version for users that have some recordings already and then merge this and release a `2.0` version.

Reply:
The better approach in my opinion is adding a delegate method that lets user compose the hash for each request based on their needs. We can have a default hashing method and keep it to prevent broken recordings and we can expose a delegate so that people can compose the hash for each request based on their project needs. What do you think @scelis?

Reply:
Instead of bumping the version, what do you think about calculating multiple hashes for backwards compatibility? When loading, if the new/updated hash doesn't match anything on disk calculate the backwards-compatible hash and check for that file, too.

Reply:
My suggestion, as I explained in another comment, is keeping the hashing code as is, so don’t breaking the previous versions, and expose an optional delegate method for people to calculate hash based on their needs because I think each project has different requirements here. For one project the headers are not important for the other one the body of the request. Some project need to remove the headers or the body from the hash to get same response on a set of request. It completely depends on the project needs.

Reply:
I think that is a decent feature request, but that MockDuck should still provide basic hash-computation functionality that should work for the 95% use case.